### PR TITLE
[25157] Style the forms of the Gitolite plugin

### PR DESCRIPTION
**Pull request by HDinger**
This styles the forms of the gitolite plugin.

https://community.openproject.com/projects/cern/work_packages/25157/activity

### CODE REVIEW

**Comment by HDinger** [comment]
### My Public Keys
#### Before
bildschirmfoto 2017-06-16 um 14 13 55

#### After
bildschirmfoto 2017-06-16 um 14 12 12

**Comment by HDinger** [comment]
### Manage gitolite repo
#### Before
bildschirmfoto 2017-06-16 um 14 13 41

#### After
bildschirmfoto 2017-06-16 um 14 11 54

File: app/views/repository_git_extras/_form.html.erb [hunk 1]
```diff
@@ -1,22 +1,19 @@
+<div id="validation_messages"><%= error_messages_for 'git_extra' %></div>
 
-    <div id="validation_messages"><%= error_messages_for 'git_extra' %></div>
+<%= f.hidden_field :git_daemon, value: (@project.is_public ? @git_extra[:git_daemon] : false), id: 'repository_git_extra_git_daemon_hidden' %>
```
Comment (HDinger) [inline app/views/repository_git_extras/_form.html.erb:3]: @machisuji Will it cause problems that I changed the ID here?

Comment (machisuji) [reply app/views/repository_git_extras/_form.html.erb:3]: I think not.

**Comment by machisuji** [comment]
If you still need to do changes (including that ID change) please do it in a new PR. I went ahead and merged the current state .

**Pull request merged**